src/translations/ru.json

```
@@ -0,0 +1,39 @@
+{
+  "log_in": "Войти",
+  "log_out": "Выйти",
+  "logged_in_as": "Вы вошли как",
+  "logged_in": "Вы вышли",
+  "logging_in": "Вхожу",
+  "logging_out": "Выхожу",
+  "sign_up": "Зарегистрироваться",
+  "signing_up": "Идет регистрация",
+  "forgot_password": "Забыли пароль?",
+  "recover_password": "Восстановить пароль",
+  "send_recovery_email": "Отправить письмо для восстановления",
+  "sending_recovery_email": "Отправляю письмо для восстановления",
+  "never_mind": "Позже",
+  "update_password": "Обновить пароль",
+  "updating_password": "Обновляю пароль",
+  "complete_your_signup": "Завершите регистрацию",
+  "site_url_title": "Настройки разработки",
+  "site_url_link_text": "Очитать локальный URL",
+  "site_url_message": "Похоже, что вы используте локальный сервер. Введите адрес вашего сайта.",
+  "site_url_label": "Адрес сайта Netlify",
+  "site_url_placeholder": "URL сайта Netlify",
+  "site_url_submit": "Установить URL сайта",
+  "message_confirm": "Электронное письмо с потверждением отправлено, нажмите на ссылку в письме для продолжения.",
+  "message_password_mail": "Мы отправили электронное письмо для восстановления, перейдите по ссылке в письме для восстановления пароля.",
+  "message_email_changed": "Ваша электронная почта обновлена!",
+  "message_verfication_error": "Ошибка верификации аккаунта. Попробуйте снова или свяжитесь с администратором.",
+  "message_signup_disabled": "Публичная регистрация закрыта. Свяжитесь с администратором для приглашения.",
+  "form_name_placeholder": "Name",
+  "form_email_label": "Введите ваш адрес электронной почты",
+  "form_name_label": "Введите имя",
+  "form_email_placeholder": "Электронная почта",
+  "form_password_label": "Введите пароль",
+  "form_password_placeholder": "Пароль",
+  "coded_by": "Сделано в Netlify",
+  "No user found with this email": "Нет пользователя с такой электронной почтой",
+  "Invalid Password": "Неправильный пароль",
+  "continue_with": "Продолжить"
```

The "continue_with" label is used in the button for authenticating with social media and other third-party services. Without the preposition, it would sound incorrect.

There are also two fields missing compared to the English version. If they are relevant, I would suggest to translate them like this:

```suggestion
  "continue_with": "Продолжить с",
  "Email not confirmed": "Адрес электронной почты не подтверждён",
  "User not found": "Пользователь не найден"
```